addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -199,7 +199,8 @@
                             </div>
 
                             <!-- CASE: converter when currency is different than the company one -->
-                            <label for="total_amount" string="" invisible="not is_multiple_currency and not product_has_cost"/>
+                            <label for="total_amount" string="Total" invisible="is_multiple_currency or not product_has_cost"/>
+                            <label for="total_amount" string="" invisible="not is_multiple_currency"/>
```

Comment:
Since there are now two labels, it would be convenient to provide names for both of them so they may be differentiable, e.g.:

`total_amount_multicurrency`

Reply:
Names have been added.